src/openrct2/management/NewsItem.cpp

```diff
@@ -484,3 +484,30 @@ void news_item_add_to_queue_custom(NewsItem* newNewsItem)
     newsItem++;
     newsItem->Type = NEWS_ITEM_NULL;
 }
+
+void news_item_remove(int32_t index)
+{
+    if (index < 0 || index >= MAX_NEWS_ITEMS)
+        return;
```

Comment:
Should we log something in case this means someone doing crazy stuff with plugins?

Reply:
No you would use `duk_error` to throw a JavaScript exception.